dashboard/src/hooks/useCurrentWorkspaceAndApplication.tsx

```diff
@@ -41,7 +42,7 @@ export function useCurrentWorkspaceAndApplication(): UseCurrentWorkspaceAndAppli
           },
         ],
         deployments: [],
-        subdomain: 'localhost',
+        subdomain: 'local',
```

Comment:
I am not sure why here we're setting the subdomain to `local` but to `localhost` here https://github.com/nhost/nhost/pull/1467/files#diff-1151af892b0f19ccb8e4e69593896b1dfcf8fd9a400a87e9d1bf0e53adb7978cR41.